charts/quantum-serverless/charts/gateway/templates/rayclustertemplate.yaml

```diff
@@ -141,28 +141,80 @@ data:
                 - name: OT_ENABLED
                   value: {{ .Values.application.ray.openTelemetryCollector.enabled | quote }}
 {{- end }}
-            - image: fluent/fluent-bit:1.9.10
-              name: ray-head-logs
+            - image: {{ .Values.application.ray.proxyImage }}
```

Comment:
I think it might be a good idea to add a Helm config option to enable / disable the proxy. I'd also make it default to disabled.